pages/people/[id]/records/[recordId]/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from 'next/router';
+
+import Seo from 'components/Layout/Seo/Seo';
+import BackButton from 'components/Layout/BackButton/BackButton';
+import CaseRecap from 'components/Cases/CaseRecap/CaseRecap';
+import PersonView from 'components/PersonView/PersonView';
+
+const CaseView = (): React.ReactElement => {
+  const {
+    query: { id, recordId },
+  } = useRouter();
+  return (
+    <>
+      <Seo title={`#${id} Case note`} />
```

Comment:
Here should be `#recordId record`. I know we have some mess around the codebase about titles, I will create a ticket